Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -1770,9 +1770,10 @@ static CGImageRef __CGContextCreateRenderableImage(CGImageRef image) {
 #pragma region Drawing Parameters - Stroke / Fill Patterns
 
 template <typename ContextStageLambda> // Takes the form HRESULT(*)(CGContextRef)
-static HRESULT _CreatePatternBrush(
-    CGContextRef context, CGPatternRef pattern, const CGFloat* components, ID2D1BitmapBrush1** brush, ContextStageLambda&& contextStage) {
-    // TODO #1592: change to support grayscale (masks) after dustins change.
+static HRESULT _CreatePatternBrush(CGContextRef context,
+                                   CGPatternRef pattern,
+                                   ID2D1BitmapBrush1** brush,
+                                   ContextStageLambda&& contextStage) {
```

Comment:
it's much nicer to have the lambda last in my opinion. 

---
In reply to: [101327104](https://github.com/Microsoft/WinObjC/pull/1952#discussion_r101327104) [](ancestors = 101327104)